WebDriverAgentLib/Categories/XCUIApplication+FBHelpers.m

```diff
@@ -216,13 +217,18 @@ + (NSDictionary *)dictionaryForElement:(id<FBXCElementSnapshot>)snapshot
       },
       FBExclusionAttributeFocused: ^{
           return [@([wrappedSnapshot isWDFocused]) stringValue];
+      },
+      FBExclusionAttributePlaceholderValue: ^{
+          return FBValueOrNull(wrappedSnapshot.wdPlaceholderValue);
```

Comment:
I'm not sure if it makes sense to have nulls everywhere as this attribute only makes sense for a limited range of elements